README.md

```diff
@@ -32,6 +32,10 @@ $ cp .env.example .env
 ```
 
 7. Don't forget to update `TODO` vars in your `.env` file!
+8. Make sure your `/etc/hosts` file contains `wpplugin.lcl.organic.ly`.
+```bash
+$ organic dev update-hosts
```

Comment:
If/once the approach in this PR is approved, I'll open and merge PR for the update to the hosts.yaml file in organic dev.